sentinel/test/unit/transitions.js

```diff
@@ -218,12 +218,6 @@ exports['attach handles missing meta data doc'] = test => {
     test.done();
   };
   transitions._attach();
-  test.equal(feed.callCount, 1);
```

Comment:
I think you've been a bit overzealous by removing all these assertions. For example, I think it's quite useful to ensure that sentinel passes the correct `since` value when resuming backlog processing. The assertions don't need to be copied in three places, and not all of the assertions are useful, but as far as I can tell your change removes all testing coverage of the backlog processing changes feed which is a fairly important part of our app.

Reply:
I presumed that there would be a test that directly tested these things, as opposed to it being bolted on to a test that tested something else.

I'll look at writing a specific test for it before merging.